app/src/androidTest/java/sdp/moneyrun/authentication/LoginInstrumentedTest.java

```diff
@@ -85,6 +88,28 @@ public void describeTo(Description description) {
         };
     }
 
+    public class ToastMatcher extends TypeSafeMatcher<Root> {
+
+        @Override
+        public void describeTo(Description description) {
+            description.appendText("is toast");
```

Comment:
Why is this necessary?

Reply:
That particularly I don't know. But the whole ToastMatcher is in the current implementation of tests useless

Reply:
alright!